packages/app/src/sandbox/eval/presets/vue-cli/index.js

```diff
@@ -151,6 +152,9 @@ export default function initialize() {
   vuePreset.registerTranspiler(m => m.path.endsWith('pug'), [
     { transpiler: pugTranspiler },
   ]);
+  vuePreset.registerTranspiler(m => m.path.endsWith('coffee'), [
+    { transpiler: coffeeTranspiler },
```

Comment:
I think it would be solved by appending

```
{ transpiler: babelTranspiler }
```

to the array!

Reply:
Ah, I will try to get this to work!

Reply:
Actually, I tried this but it didn't work. Instead, I had to add `const postLoaders = { coffee: 'babel-loader' };` to `transpilers/vue/loader.js`, which made it all work.

Reply:
I tried this, but it didn't work. I added `const postLoaders = { coffee: 'babel-loader' };` to `eval/transpilers/vue/loader.js` and it worked!